extractor/src/main/java/org/schabi/newpipe/extractor/services/soundcloud/SoundcloudPlaylistExtractor.java

```diff
@@ -64,21 +74,20 @@ public String getThumbnailUrl() {
             // if it also fails, return null
             try {
                 final InfoItemsPage<StreamInfoItem> infoItems = getInitialPage();
-                if (infoItems.getItems().isEmpty()) return null;
 
                 for (StreamInfoItem item : infoItems.getItems()) {
-                    final String thumbnailUrl = item.getThumbnailUrl();
-                    if (thumbnailUrl == null || thumbnailUrl.isEmpty()) continue;
-
-                    String thumbnailUrlBetterResolution = thumbnailUrl.replace("large.jpg", "crop.jpg");
```

Comment:
Why did you remove the code for better resolution? Does api-v2 return that by default?

Reply:
I didn't remove it, I just moved it below

Reply:
Sorry for not noticing this, didn't I tell you I was tired?